bika/lims/browser/templates/stickers_preview.pt

```diff
@@ -241,11 +294,10 @@
               </tal:tick>
             </div>
             <tal:stickers repeat="sticker view/items">
-            <tal:sticker define="item_id python:sticker[2].getId() if sticker else None;"
-                         condition="python:not item_id or item_id not in view.rendered_items">
-            <div class='sticker'
-                 tal:content='structure python:view.renderItem()'></div>
-            </tal:sticker>
+                    <tal:sticker define="item_id python:sticker[2].getId() if sticker else None;">
```

Comment:
I tried to figure out about that `view.rendered_items`. I understand why you need to remove it, cause each time the function `nextItem` is called, the current item is added in this `view.rendered_items` list, preventing you to display a label for a given object more than once. The thing is that I am not sure for what `view.rendered_items` var is used other than this condition... Anyhow, add a `condition` for at least check if `item_id` is not `None` (note that in `define` attribute there is the possibility to set a `none` value to `item_id`, which is something weird again...)
```

Reply:
@xispa I also checked the use of `rendered_items` and agree in that it was only used in the check that I removed to be able to print more than one copy. I am then going to remove `rendered_items` from the code.